augurpy/cv.py

```diff
@@ -30,31 +30,36 @@ def run_cross_validation(
     estimator: Union[RandomForestRegressor, RandomForestClassifier, LogisticRegression],
     subsample_idx: int,
     folds: int = 3,
+    random_state=Optional[int],
 ) -> Dict:
     """Perform cross validation on given subsample.
 
     Args:
         subsample: subsample of gene expression matrix of size subsample_size
         estimator: classifier object to use in calculating the area under the curve
-        folds: number of folds
         subsample_idx: index of subsample
+        folds: number of folds
+        random_state: set random seed
 
     Returns:
         Dictionary containing prediction metrics and estimator for each fold.
     """
     scorer = set_scorer(estimator)
     x = subsample.to_df()
     y = subsample.obs[[col for col in subsample.obs if col.startswith("y")]]
+    cv = KFold(n_splits=folds, random_state=random_state, shuffle=True)
```

Comment:
Mhm, I liked the variable name folds that you passed before better.

A KFold object is not a cross validation :)

Reply:
done:)